lib/date.js

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment';
+
+export function exhibitionPeriod(startAt, endAt) {
+  const startMoment = moment(startAt);
+  const endMoment = moment(endAt);
+  const thisMoment = moment();
+
+  let startFormat = 'MMMM Do';
+  if (startMoment.year() !== endMoment.year()) {
+    startFormat = startFormat.concat(', YYYY');
+  }
+
+  let endFormat = 'Do';
+  if (endMoment.year() !== thisMoment.year()) {
+    endFormat = endFormat.concat(', YYYY');
+  }
+  if (!(startMoment.year() === endMoment.year() && startMoment.month() === endMoment.month())) {
+    endFormat = 'MMMM '.concat(endFormat);
+  }
+
+  return `${startMoment.format(startFormat)} - ${endMoment.format(endFormat)}`;
+}
```

Comment:
> In general I've been shying away from including predetermined date formatting in the schema

Yeah, I do get that, but we just keep re-implementing these things. So I’m thinking that maybe it’s time for that shift in moving view models into the ☁️ ?

> I'd like to avoid the nonsense we have in Force though where there are many subtly different date string formatting methods on models.

I’m not familiar with the Force code-base, but did have a hard time figuring out all the date methods when doing some cursory reading for this work.

I agree that I’d also like to avoid that, so don’t hold back when reviewing!